tests/phpunit/integration/Core/Notifications/NotificationsTest.php

```diff
@@ -42,4 +49,44 @@ function ( REST_Route $route ) {
 			$routes
 		);
 	}
+
+	public function test_non_proxy_request_does_not_request_proxy() {
+
+		$context                        = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE, new MutableInput() );
```

Comment:
No need for `MutableInput` here as we aren't mutating any super global input arrays.
```suggestion
		$context                        = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
```